### PR TITLE
Modified the readme about how to compile in Windows

### PR DESCRIPTION
**Pull request by alexxucui**
Tested in Windows and works well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Self-Driving Car Engineer Nanodegree Program
 
 1. Clone this repo.
 2. Make a build directory: `mkdir build && cd build`
-3. Compile: `cmake .. && make`
+3. Compile: `cmake .. && make` (Windows: `cmake .. -G "Unix Makefiles" && make`)
```
Comment (cameronwp) [inline README.md]: Nit: could we just make this a subbullet of step 3? Eg.

3. Compile: ...
  * On Windows, you may need to run: ...

**Comment by cameronwp** [comment]
One nit to fix, but LGTM! Thanks for submitting :)

**Comment by alexxucui** [comment]
That looks good! Thanks!

**Comment by cameronwp** [comment]
Heyo, sorry if I wasn't clear before. I was asking for you to amend your commit to the format given. As soon as you do that, I'll merge. Thanks!

**Comment by alexxucui** [comment]
Done.

**Comment by cameronwp** [comment]
Beautiful! Thanks for doing that :)

**Pull request merged**